security/pkg/nodeagent/vm/config.go

```diff
@@ -21,6 +21,43 @@ import (
 	"istio.io/istio/security/pkg/caclient"
 )
 
+// Set of environment variables for VM Citadel Agent
+const (
+	// Name of the org for the cert.
+	OrgName = "ISTIO_CA_ORG_NAME"
```

Comment:
One suggestion. The naming of the var should be reflected the fact that they're environment variable name.

Either change these to OrgEnvName, or put all into a separate envname package, such that it can be referenced as envnamr.Org, etc.

Reply:
You can also look up other component, pilot,mixer etc, implementation to look for better practice.

Reply:
Sg. I created a package called `envvar` under `nodeagent`, maybe later we can refacter k8s citadel agent to use this.